Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Rebuild the source code only when needed
+FROM registry.access.redhat.com/ubi8/nodejs-14-minimal AS builder
+USER root
+WORKDIR /app
+RUN npm install -g yarn
+COPY . .
+RUN $(npm get prefix)/bin/yarn install --frozen-lockfile --ignore-optional
+RUN $(npm get prefix)/bin/yarn format
+RUN $(npm get prefix)/bin/yarn build
+RUN $(npm get prefix)/bin/yarn install --production --ignore-scripts --prefer-offline --ignore-optional
+
+# Production image, copy all the files and run next
+FROM registry.access.redhat.com/ubi8/nodejs-14-minimal AS runner
```

Comment:
Clair (the security scanner on quay) is reporting a High severity vulnerability with nodemon. Not sure if it's in the base image, or something that gets added with the viewer, but we might want to update it:
<img width="1667" alt="Screen Shot 2021-07-22 at 3 26 34 PM" src="https://user-images.githubusercontent.com/6880023/126698191-f992061e-2883-489c-8be9-4f2d580c3334.png">

Reply:
It is not the registry viewer becasue nodemon is not currently inside of package.json.

Reply:
Weird

Reply:
Probably in the base image then.